src/opcode.rs

```diff
@@ -1600,19 +1600,29 @@ opcode!(
 
 opcode!(
     /// Send a zerocopy message on a socket, equivalent to `send(2)`.
+    ///
+    /// A fixed (pre-mapped) buffer can optionally be used from pre-mapped buffers that have been
+    /// previously registered with [`Submitter::register_buffers`](crate::Submitter::register_buffers).
     pub struct SendZc {
+        /// The `buf_index` is an index into an array of fixed buffers,
```

Comment:
Should the document be placed on `buf_index` field?

Reply:
Done. I had taken my cue from the ReadFixed example. Should I issue a PR to move that field's comment?